google/resource_google_project.go

```diff
@@ -52,8 +52,13 @@ func resourceGoogleProject() *schema.Resource {
 			},
 			"org_id": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Optional: true,
+				Computed: true,
+			},
+			"folder_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
If you specify a folder, you should not be able to specify an org. To make sure it fails at 
 the `terraform plan` step, add `ConflictsWith: []string{"org_id"}` in the schema definition for the `folder_id` field. And vice-versa

Reply:
Done.